tensorboard/webapp/hparams/_types.ts

```diff
@@ -33,21 +33,3 @@ export interface HparamAndMetricSpec {
   hparams: HparamSpec[];
   metrics: MetricSpec[];
 }
-
-export interface DiscreteFilter {
-  type: DomainType.DISCRETE;
-  includeUndefined: boolean;
-  possibleValues: DiscreteHparamValues;
-  // Subset of `possibleValues`
-  filterValues: DiscreteHparamValues;
-}
-
-export interface IntervalFilter {
-  type: DomainType.INTERVAL;
-  includeUndefined: boolean;
-  minValue: number;
-  maxValue: number;
-  // Filter values have to be in between min and max values (inclusive).
-  filterLowerValue: number;
-  filterUpperValue: number;
-}
```

Comment:
These have been moved to the data  table types because that is where the filtering is being done now.